### ISSUE TITLE
Enchanting problem

### ISSUE THREAD

**Issue by cybrann** [JadeCore/JadeCore#68]
Guys enchanting is bugged on 5.4.8 branch and I've tried to fix it for 2-3 hours with no luck. Problem is whenever I try to enchant an item it gives me the error: Tried to enchant an item that didn't exist.

**Comment by Bodeguero** [comment]
@cybrann all items are like this? Or only the item you want to enchant? You could pass me this Item ID?

**Comment by phulo** [comment]
seems

**Comment by cybrann** [comment]
All items.

**Comment by phulo** [comment]
i can upload my commits to the repo, branch 548 why?

**Comment by cybrann** [comment]
@phulo Please do not post unrelated things on issue reports.

**Comment by phulo** [comment]
write me im in jadecore  team on skype

**Comment by wowharley** [comment]
yes i've notice the same things with the items not enchanting, after i'm done with what i'm doing i'll take a look at the source.

**Comment by cooler-SAI** [comment]
@phulo , second chance - make few commits if you are in the Team

**Comment by cybrann** [comment]
@cooler-SAI He had problems with access, I fixed it 2-3 mins ago.
